### PR TITLE
PTRENG-4192 code cleanup v2

### PR DESCRIPTION
**Pull request by chb0github**

### CODE REVIEW

**Review by alexhung — CHANGES_REQUESTED** [review]
All looks good except the one test using regex with a typo.

File: pkg/artifactory/resource/security/resource_artifactory_scoped_token_test.go [hunk 1]
```diff
@@ -338,7 +339,7 @@ func TestAccScopedToken_WithTooLongAudiences(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config:      scopedTokenConfig,
-				ExpectError: regexp.MustCompile(".*Total combined length of audiences field exceeds 255 characters:.*"),
+				ExpectError: regexp.MustCompile(".*Total combined length of audences field exceeds 255 characters:.*"),
```
[the remark this answers is not in the document]
Comment (chb0github) [reply pkg/artifactory/resource/security/resource_artifactory_scoped_token_test.go:342]: I was gun shy after last run. I'll fix it

File: pkg/artifactory/resource/security/resource_artifactory_scoped_token_test.go [hunk 2]
```diff
@@ -338,7 +339,7 @@ func TestAccScopedToken_WithTooLongAudiences(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config:      scopedTokenConfig,
-				ExpectError: regexp.MustCompile(".*Total combined length of audiences field exceeds 255 characters:.*"),
+				ExpectError: regexp.MustCompile(".*total combined length of audiences field exceeds 255 characters:.*"),
```
Comment (alexhung) [inline pkg/artifactory/resource/security/resource_artifactory_scoped_token_test.go:342]: Is the lowercase 'total' intentional? I'm not seeing a case-insensitive regex here. If this test passes, then all's good.

Comment (chb0github) [reply pkg/artifactory/resource/security/resource_artifactory_scoped_token_test.go:342]: Yes it is and yes I fixed the test. Golang spec says that error strings should be lower case

**Review by alexhung — APPROVED** [review]

**Pull request merged**